.travis.yml

```diff
@@ -2,12 +2,13 @@ language: python
 
 python:
   - "3.6"
-  - "3.7-dev"
+  - "3.7"
+  - "3.8"
 
 env:
   - DEPS="Django>=1.11,<1.12"
-  - DEPS="Django>=2.0,<2.1"
```

Comment:
Not sure if we should drop Django 2.0 support now.

Reply:
Support of Django 2.0 has ended on April 1, 2019 and Django 2.1 on December 2, 2019. Do you think testing those versions are still useful? Maybe just keep 2.1 version in that case?

Reply:
Good point but we should mention that in the CHANGELOG then. :)

Reply:
Done!